backend/src/common/enums/app/env.enum.ts

```diff
@@ -12,6 +12,9 @@ const {
   DB_POOL_MAX,
   DB_DIALECT,
   SECRET_KEY,
+  UDEMY_CLIENT_ID,
+  UDEMY_CLIENT_SECRET,
+  UDEMY_API_BASE,
```

Comment:
```suggestion
  UDEMY_BASE_URL,
```
don't forget to add this ENVs to `env.example`